packages/guides-restructured-text/src/RestructuredText/Parser/Productions/TransitionRule.php

```diff
@@ -87,9 +87,9 @@ public function apply(DocumentParserContext $documentParserContext, ?CompoundNod
         return new SeparatorNode(1);
     }
 
-    public function isSpecialLine(string $line): ?string
+    public function isSpecialLine(string $line, int $minimumLength = 2): ?string
```

Comment:
I think this method is never used by any other class. If not, please make it private :-D

Reply:
It is duplicated in Transition and title, however, so I extracted it